tests/test_acasclient.py

```diff
@@ -2352,6 +2353,191 @@ def test_50_delete_ls_thing(self):
+        self.registerTestSalt() 
+        search_result = self.client.search_salts("Benzoic Acid")
+        id = search_result[0]["id"]
+        updated_struct = "\n  Ketcher 08172213572D 1   1.00000     0.00000     0\n\n 11 11  0     1  0            999 V2000\n    5.2250   -4.6500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    6.0910   -5.1500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    6.0910   -6.1500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    5.2250   -6.6500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    4.3590   -6.1500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    4.3590   -5.1500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    6.9570   -4.6500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    7.8231   -5.1500    0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0\n    6.9570   -3.6500    0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0\n    7.8231   -6.1500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    8.6891   -6.6500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n  1  2  1  0     0  0\n  2  3  2  0     0  0\n  3  4  1  0     0  0\n  4  5  2  0     0  0\n  5  6  1  0     0  0\n  6  1  2  0     0  0\n  2  7  1  0     0  0\n  7  8  1  0     0  0\n  7  9  2  0     0  0\n  8 10  1  0     0  0\n 10 11  1  0     0  0\nM  END\n"
+        updated_name = "Ethyl benzoate"
+        updated_abbrev = "BzCOOCH2CH3"
+        dryrun = "false"
+        edit_response = self.client.edit_salt(id, dryrun, updated_abbrev, updated_name, updated_struct)
+        self.assertIn("The abbreviation of this salt will be changed.", edit_response[0]["message"])
+        self.assertIn("The name of this salt will be changed.", edit_response[1]["message"])
+        self.assertIn("The weight of this salt will be changed.", edit_response[2]["message"])
+        search_result = self.client.search_salts("Ethyl benzoate")
+        self.assertIn("BzCOOCH2CH3", search_result[0]["abbrev"])
+        id = search_result[0]["id"]
+        delete_response = self.client.delete_salt(id)
+        self.assertIn("There were no lot dependencies found for this salt.", delete_response)
+
+    def test_62_edit_salt_check_salt_same_name_or_abbrev(self):
+        self.removeAllSalts()
+        # Register Benzoic Acid 
+        self.registerTestSalt() 
+        # Register Iodine 
+        dryrun = "false"
+        abbrev = "I"
+        name = "Iodine"
+        mol_structure = "\n  Ketcher 08172213312D 1   1.00000     0.00000     0\n\n  1  0  0     1  0            999 V2000\n    7.4500   -7.1750    0.0000 I   0  0  0  0  0  0  0  0  0  0  0  0\nM  CHG  1   1  -1\nM  END\n"
+        self.client.create_salt(abbrev, name, mol_structure, dryrun)
+        # Try to Change Benzoic Acid Abbreviation and Name to Iodine's Abbreviation and Name 
+        search_result = self.client.search_salts("Benzoic Acid")
+        id = search_result[0]["id"]
+        updated_struct = search_result[0]["molStructure"]
+        self.assertIn("INDIGO", updated_struct)
```

Comment:
nit: This step might fail if we're using a different chemistry engine. Maybe just check that the molStructure is non-empty?